lib/libzfs_core/libzfs_core.c

```diff
@@ -169,6 +171,29 @@ lzc_ioctl(zfs_ioc_t ioc, const char *name,
 	return (error);
 }
 
+static int
+lzc_ioctl(const char *cmd, const char *name, nvlist_t *source,
```

Comment:
@ahrens My apologies for missing this comment. As @trisk said, the idea is to avoid mixing configuration from data.